packages/cli/src/ActiveExecutions.ts

```diff
@@ -59,32 +66,37 @@ export class ActiveExecutions {
 			const execution = ResponseHelper.flattenExecutionData(fullExecutionData);
 
 			const executionResult = await Db.collections.Execution.save(execution as IExecutionFlattedDb);
+			// TODO: what is going on here?
```

Comment:
This is probably legacy from when we supported MongoDB where IDs where instances of `ObjectID`

Reply:
So.... should we keep it in?

Reply:
I guess we could remove this, since this is in the codebase for over 2 years.